lib/gooddata/lcm/actions/synchronize_user_groups.rb

```diff
@@ -27,31 +27,44 @@ class SynchronizeUserGroups < BaseAction
 
         description 'Additional Hidden Parameters'
         param :additional_hidden_params, instance_of(Type::HashType), required: false
+
+        description 'Disable dashboard permission for pixel perfect dashboard'
```

Comment:
Disable synchronizing dashboard permissions for Pixel Perfect dashboards.
Disable synchronizing dashboard permissions for AD/KD dashboards.